index.d.ts

```diff
@@ -407,6 +408,35 @@ type HeadlineLink = {
     preventFocus?: boolean; // if true, stop the link from being tabbable and focusable
 };
 
+type AnswerType = {
+    id: string;
+    text: string;
+    revealText?: string;
+    isCorrect: boolean;
+    answerBuckets: string[];
+};
+
+type QuizAtomResultBucketType = {
+    id: string;
+    title: string;
+    description: string;
+    // assets TODO:
+}
+
+type QuestionType = {
+    id: string;
+    text: string;
+    answers: AnswerType[];
+    imageUrl?: string;
+    resultBuckets?: QuizAtomResultBucketType[]
+}
+
+type ResultBucketsType = {
+    id: string;
+    title: string;
+    description: string;
+}
+
```

Comment:
I think the definitions are a bit loose but maybe these would live better in `content.d.ts`? This might just be me but I've always thought of content as stuff around elements and `index` as more general stuff.